backend/proxy/README.md

```diff
@@ -31,6 +52,8 @@ Now you can sign into Container Registry.
 $ heroku container:login
 ```
 
+_Note_ if you are using an Apple M1 device the following steps will not work. You will have to do the following: https://stackoverflow.com/a/67001433
```

Comment:
@ludtkemorgan , I have an M1, and I don't think I've had to do this in the past, FYI.

Reply:
@seanmalbert I ran the steps listed in this README again and I got the same error again. When it deploys to Heroku the logs show this:
```
Apr 24 10:31:38 [bloom-backend-main-proxy](https://my.papertrailapp.com/systems/bloom-backend-main-proxy/events?focus=1587209679106510850&selected=1587209679106510850) [heroku/web.1](https://my.papertrailapp.com/events?q=program%3Aheroku%2Fweb.1&focus=1587209679106510850&selected=1587209679106510850) State changed from starting to crashed
Apr 24 10:31:38 [bloom-backend-main-proxy](https://my.papertrailapp.com/systems/bloom-backend-main-proxy/events?focus=1587209679222063112&selected=1587209679222063112) [app/web.1](https://my.papertrailapp.com/events?q=program%3Aapp%2Fweb.1&focus=1587209679222063112&selected=1587209679222063112) Error: Exec format error
Apr 24 10:31:38 [bloom-backend-main-proxy](https://my.papertrailapp.com/systems/bloom-backend-main-proxy/events?focus=1587209679268962304&selected=1587209679268962304) [heroku/web.1](https://my.papertrailapp.com/events?q=program%3Aheroku%2Fweb.1&focus=1587209679268962304&selected=1587209679268962304) Process exited with status 126
```